src/stylesheets/custom.scss

```diff
@@ -37,3 +37,14 @@
 .usa-header {
   font-family: 'Public Sans';
 }
+
+.usa-footer {
+  // Ideally, this width should be set in _settings.scss, alongside the other variables
```

Comment:
https://github.com/uswds/uswds/issues/3532

We might have to updated USWDS for this solution. I'll look into this after it gets merged.